pyproject.toml

```diff
@@ -28,8 +28,8 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 # Used for local development
-nautobot = "^1.6.0"
-diffsync = "^1.5.1"
+nautobot = "^2.0.0"
```

Comment:
@glennmatthews would we want to have the minimum supported version be 2.0.6 due to the recent vulnerabilities & we haven't done a v2 supported release yet?

Reply:
Good question. I'd be in favor of that as a guideline across the apps to be honest :-)

Reply:
Updated minimum Nautobot version to 2.0.6